daemon/main.go

```diff
@@ -15,6 +15,8 @@
 package main
 
 import (
+	_ "github.com/cilium/cilium/pkg/debugdetection"
```

Comment:
I see what you did here. You can move this to line 30 and it will be okay as well. No? Also, right a comment before this line explaining why we are doing this.